package.json

```diff
@@ -58,6 +58,7 @@
     "jest-watch-typeahead": "2.2.2",
     "lint-staged": "15.2.10",
     "prettier": "3.3.3",
+    "prettier-jest": "npm:prettier@^2",
```

Comment:
Newest version of prettier does not work with Jest so we need to use an older one.
Referece: https://jestjs.io/docs/configuration/#prettierpath-string